source/octf/cli/internal/param/ParamString.h

```diff
@@ -3,17 +3,23 @@
  * SPDX-License-Identifier: BSD-3-Clause-Clear
  */
 
-#ifndef SOURCE_OCTF_CLI_PARAM_PARAMSTRING_H
-#define SOURCE_OCTF_CLI_PARAM_PARAMSTRING_H
+#ifndef SOURCE_OCTF_CLI_INTERNAL_PARAM_PARAMSTRING_H
+#define SOURCE_OCTF_CLI_INTERNAL_PARAM_PARAMSTRING_H
 
 #include <string>
-#include <octf/cli/CLIElement.h>
-#include <octf/cli/param/Parameter.h>
+#include <octf/cli/internal/CLIElement.h>
+#include <octf/cli/internal/param/Parameter.h>
 
 namespace octf {
+namespace cli {
 
 constexpr int MAX_STRING_LENGTH = 4096;
 
+/**
+ * This value specifies delimiter for multiple value parameters
+ */
+constexpr char PARAMETER_VALUE_DELIMITER[] = ",";
```

Comment:
I'm thinking if we will use other 'multiple' parameters in the future (for example multiple enum values). Then it would be better to put this in some common file (ICommand.h maybe?)

Reply:
So then we will publish it